src/main/java/ai/datahunters/md/server/analytics/AnalyticsService.java

```diff
@@ -0,0 +1,22 @@
+package ai.datahunters.md.server.analytics;
+
+import ai.datahunters.md.server.photos.PhotosRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.CompletableFuture;
+
+@Service
+public class AnalyticsService  {
+
+    private PhotosRepository repo;
+
+    @Autowired
+    public AnalyticsService(PhotosRepository repo) {
```

Comment:
if you are using this `AllArgsConstructor` please be consistent and use it here as well